include/morphio/vector_types.h

```diff
@@ -7,14 +7,20 @@
 #include <vector>
 
 namespace morphio {
-using Point = std::array<float, 3>;
+#ifdef USE_DOUBLE
+using floatType = double;
```

Comment:
what about `long double`?

Reply:
NEURON uses double. But if needed one can adjust the code once again :D